docs/src/administration/users.md

```diff
@@ -1,138 +1,162 @@
 
-## Manage user permissions in the console
+| Environment permission   | Description |
+|------------------------- |-------------|
+|Viewer                    | Viewers can view all environments of this type.|
+|Contributor               | Contributors can push code and branch all environments of this type. This role includes the Viewer permissions.
+|Admin                     | Admins can change settings and execute actions on all environments of this type. This role includes the Viewer and Contributor permissions.
 
-From your list of projects, select the project where you want to view or edit user permissions. Click the Settings tab at the top of the page, then click the `Access` tab on the left to see the users in the project and their roles.
+## Manage users
 
-![Project user management screenshot](/images/management-console/settings-project-access.png)
+### Add a user to a project
 
-Selecting a user to edit that user's permissions or delete the user's access to the project entirely.
+To add a user to a project, follow these steps:
 
-To add a new user, click the `Add` button.
+{{< codetabs >}}
+---
+title="In the console"
+---
+1. Go to your console and select the project where you want to add a new user.
+2. Under **Settings**, click **Access**.
+1. Click **+ Add**
+1. Add the user's details and choose their permissions.
+4. Click **Save**.
+5. After you add a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by clicking **Redeploy** in the console.
+---
+title="Using the CLI"
+---
+You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to create new users.
 
-You can either grant the `Project admin` role to the user, which will give them `Admin` access to every environment in the project, or grant specific permissions for each environment type.
+* `platform user:add`
 
-After inviting a new user, users without a Platform.sh account receive an invitation email asking to confirm their details and register an account.
+Say you want to add `user1@example.com` to the project with **Project Admin** permissions:
 
-In order to push and pull code (or to SSH into one of the project's environments), the user needs to add an SSH key or use the Platform CLI.
+```bash
+platform user:add user1@example.com -r admin
+```
+Once you add the user to the project, they receive an invitation email to join the project.
 
-If the user already has an account, they receive an email with a link to access the project.
+Use `platform list` to get the full list of commands.
 
-## Manage users permissions with the CLI
+{{< /codetabs >}}
 
-You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md). to fully manage your users and integrate with any automated system.
+Once you add a user to a project, they receive an invitation to join the project.
```

Comment:
I think it's worth noting that new users get an invitation to create an account. But maybe it's clear enough in the emails themselves? :shrug: